Automattic/jetpack
# Protect: Fix overwritten FirewallSubheading updates

Fixes #
When the WAF StatCards PR was merged we overwrote changes to the component that affect the display of accurate content.

## Proposed changes:

* Reapply changes from prior to the merge

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1448

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch on JT and activate Protect with an upgrade
* Toggle the various WAF settings and ensure that the FirewallHeader subheading content appropriately reflects the changes
* Downgrade and ensure the upgrade prompt is preset and functions accordingly
* Ensure no regressions are introduced
*
